lib/index.js

```diff
@@ -91,49 +127,78 @@ module.exports = {
             results = await batchRequest(config.global.batchSizes.instructors, config.global.pageSizes.instructors, {term: term}, 'get_instructor', this.School);
             instructors.push(...results);
         } while (instructors.length > 0 && results.length == config.global.batchSizes.instructors);
-        
+
         return instructors;
     },
-    
-    async getCampus(){         
+    /**
+     *
+     * @returns {Promise<any>}
+     */
+    async getCampus(){
         let res = await bannerRequest(this.School, 'get_campus');
-        return res.Data;
+        return res.Body;
     },
-    
+    /**
+     *
+     * @returns {Promise<any>}
+     */
     async getColleges(){
         let res = await bannerRequest(this.School, 'get_college');
-        return res.Data;
+        return res.Body;
     },
-    
-    async getAttributes(){           
+    /**
+     *
+     * @returns {Promise<any>}
+     */
+    async getAttributes(){
         let res = await bannerRequest(this.School, 'get_attribute');
-        return res.Data;
+        return res.Body;
     },
-
+    /**
+     *
+     * @returns {Promise<any>}
+     */
     async getSessions(){
         let res = await bannerRequest(this.School, 'get_session');
-        return res.Data;
+        return res.Body;
     },
 
     async getPartsOfTerm(){
         let res = await bannerRequest(this.School, 'get_partOfTerm');
-        return res.Data;
+        return res.Body;
     },
-
+    /**
+     *
+     * @returns {Promise<any>}
+     */
     async getInstructionalMethods(){
         let res = await bannerRequest(this.School, 'get_instructionalMethod');
-        return res.Data;
+        return res.Body;
     },
-
+    /**
+     *
+     * @param term
+     * @param crn
+     * @returns {Promise<any[]|BigUint64Array|Uint8ClampedArray|Uint32Array|Blob|Int16Array|Float64Array|SharedArrayBuffer|string|Uint16Array|ArrayBuffer|Int32Array|Float32Array|BigInt64Array|Uint8Array|Int8Array>}
```

Comment:
What's with all these return types? It should just be `string`

Reply:
Resolved